### PR TITLE
Add JsonResponse Upsell check

### PR DESCRIPTION
**Pull request by ulziibay**
Addresses  https://github.com/returntocorp/sgrep-rules/issues/91
Testing with `./test.py -v -sv -s python/django`

### CODE REVIEW

File: python/smells/dict-modify-iterating.yaml [hunk 1]
```diff
@@ -10,6 +10,12 @@ rules:
               for $KEY in $DICT.keys():
                   ...
                   del $DICT[$KEY]
+          - pattern: |
+              for $KEY, $VALUE in $DICT.items():
+                  del $DICT[$KEY]
+          - pattern: |
+              for $KEY in $DICT.keys():
+                  del $DICT[$KEY]
```
Comment (ievans) [inline python/smells/dict-modify-iterating.yaml]: @ulziibay did you accidentally commit this?

Comment (ulziibay) [reply python/smells/dict-modify-iterating.yaml]: No, they were failing CI

Comment (ulziibay) [reply python/smells/dict-modify-iterating.yaml]: What is going on here? This fixes the CI @ievans

**Comment by DrewDennison** [comment]
@ulziibay fixed CI problems this should be g2g

**Pull request merged**